### PR TITLE
Tableau update

### PR DESCRIPTION
**Pull request by ConnorJLN**
updated tableau to include a section on stories, and added to clarifying images.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -165,14 +165,23 @@ Now we can create a dashboard to view all of our visualizations at once.
...
 18. The text will automatically appear in the top left corner. Use the menu (click the down arrown at the top of the box) to make the text box floating, select the floating order bring to front, and reposition it. 
 19. We also want to add some instructions for using the year search box. Let's hide the year label to give us more space. Click the menu on the year box and select show Title. 
 20. Now, add an addition text for instructions. Double click on text under Objects in the bottom left corner. Enter your instructions. Make the box floating and bring to front. Then, position it where you would like. 
 21. You may also want to add an addition box to give yourself credit as the creator of this visualization. 
 
 [Return to Top](#about)
 
+## Creating A Story Page
+1. Creating a story in Tableau allows you to combine you worksheets, dashboards, and text into presentation style pages, we'll create one here.
```
Comment (sclayton29) [inline README.md]: Run-on sentence here.

File: README.md [hunk 2]
```diff
@@ -125,7 +125,7 @@ Tableau allows you to perform basic statistical analysis. Let's add a trend line
...
-7. To do this, click the dropdown menu in the upper right hand corner of the "Measure Values" card. Click the first option, "New Calculation." ![Add Calculation to Measure Values](images/TableauBarchartImg01.png) We want to aggregate the sums of the two definitions of rape. This is as simple as adding "+" between "SUM(item_1)" and "SUM(item_2)". Tableau accepts the addition operator, but will not accept SUMs of SUMs. When you enter this calculation, Tableau will change the addition operator to an aggregation "AGG()", though this is not a selectable parameter in advance.
+7. To do this, click the dropdown menu in the upper right hand corner of the "Measure Values" card. Click the first option, "New Calculation." ![Add Calculation to Measure Values](images/TableauBarchartImg01.png) We want to aggregate the sums of the two definitions of rape. This is as simple as adding "+" between "SUM(item_1)" and "SUM(item_2)". Tableau accepts the addition operator, but will not accept SUMs of SUMs. When you enter this calculation, Tableau will change the addition operator to an aggregation "AGG()", though this is not a selectable parameter in advance. ![FormulaLook](images/FigFormula.PNG)
```
Comment (sclayton29) [inline README.md]: I thought you were going to mention that null values will appear if you have a filter on for years before the updated definition.

**Review by sclayton29 — CHANGES_REQUESTED** [review]
I made a few comments on your changes for you to review. 

Other than those comments, please add your name at the top of the README file. You can add it as Updated by Connor Neale. Also, please modify the updated date.

**Comment by sclayton29** [comment]
Thanks, for the updates! I'll merge it into the main DSL lesson now.

**Comment by ConnorJLN** [comment]
Just sent those updates, let me know if I missed anything.

**Pull request merged**